parquet-hadoop/src/test/java/org/apache/parquet/hadoop/TestParquetFileWriter.java

```diff
@@ -425,6 +436,27 @@ public void testConvertToThriftStatistics() throws Exception {
   }
 
   @Test
+  public void testStatisticsGettingIgnoredFor160Version() throws Exception {
+    long[] longArray = new long[] {39L, 99L, 12L, 1000L, 65L, 542L, 2533461316L, -253346131996L, Long.MAX_VALUE, Long.MIN_VALUE};
+    LongStatistics parquetMRstats = new LongStatistics();
```

Comment:
The stats are 0 for LongStatistics because the method returns a primitive, I think. But it scares me that the API allows an empty stats object to be used like this. At a minimum, if the stats aren't set those should through an Exception. I'd prefer not to deserialize a stats object at all and return null. Is there something preventing that?

This should also not be passing tests because LongStatistics shouldn't be filtered. We should clone this and add a test to verify that BinaryStatistics are actually filtered.